app/Resources/views/procuration_manager/_proposals_list.html.twig

```diff
@@ -4,10 +4,22 @@
             {{ proxy.id }}
         </td>
         <td class="datagrid__table__col--left">
-            {{ proxy.firstNames }} {{ proxy.lastName }}
+            <strong>{{ proxy.firstNames }} {{ proxy.lastName }}</strong>
             {% if proxy.reliabilityDescription %}
                 ({{ proxy.reliabilityDescription }})
             {% endif %}
+            <br/>
+            Email :
+            <a href="mailto:{{ proxy.emailAddress }}">
+                {{ proxy.emailAddress }}
+            </a>
+            {% if proxy.phone %}
+                <br />
+                Téléphone :
+                <a href="{{ proxy.phone|phone_number_format(constant('\\libphonenumber\\PhoneNumberFormat::RFC3966')) }}">
```

Comment:
what about create a global variable to store this constant and use it in everywhere ?